docs/platform/howto/access-service-logs.rst

```diff
@@ -0,0 +1,24 @@
+Access service logs
+====================
+
+Accessing logs from Aiven service: PostgreSQL®, Kafka®, Cassandra®, Redis®*, InfluxDB®
```

Comment:
What about the other services? can I access the log for M3, flink and all the missing ones?

Reply:
I'm removing the line as well as all services will have logs.